src/plugin.ts

```diff
@@ -16,6 +53,7 @@ export async function startStopTask(inputs: PluginInputs, env: Env) {
     eventName: inputs.eventName,
     payload: inputs.eventPayload,
     config: inputs.settings,
+    organizations: [],
```

Comment:
This should not be here but in the configuration.

Reply:
I think the reasoning is that it's needed at multiple places so they just put it in the context

Reply:
The spec  states:
> Only check assigned issues within the network, but this should be configurable to either be just within the repo, org, or network

Shouldn't this be configurable? And even if not, putting it in the config also allows to add defaults on decode which would ensure it is properly initialized and types as well.

Reply:
maybe I'm missing something, but the config item `assignedIssueScope` can be set to `repo`, `org` or `network`, this `configuration` property gets filled based on `assignedIssueScope` when the plugin starts